config/lib/seed.js

```diff
@@ -92,3 +65,85 @@ function reportError(err) {
   console.log(err);
   console.log();
 }
+
+function performSeedUser(user) {
+
+  var User = mongoose.model('User');
+
+  return new Promise(function(resolve, reject) {
```

Comment:
@ilanbiala I chose to remove spaces between function arguments and the function keyword.